ComponentKit/LayoutComponents/CKStackLayoutComponent.h

```diff
@@ -78,9 +78,9 @@ struct CKStackLayoutComponentChild {
   /** Additional space to place after the component in the stacking direction. */
   CGFloat spacingAfter;
   /** If the sum of childrens' stack dimensions is less than the minimum size, should this component grow? */
-  BOOL flexGrow;
+  NSInteger flexGrow;
```

Comment:
Migrating to the new type is actually quite simple. `CKStackLayoutComponent` will continue to specify a default of _do not flex_ (a `.flexGrow` of 0 and a `.flexShrink` of 0) for the numeric type. `NO` will presumably continue to evaluate to 0 and `YES` will follow suit and evaluate to 1.

Existing callsites can continue to use `YES` or `NO` while folks more interested in more complex flex behaviors can use a proper flex factor instead. That being said it probably makes sense to do a codemod and ensure everybody is using proper integer values for these two members.